source/indexes/atlas-search-index.txt

```diff
@@ -0,0 +1,274 @@
+.. _cpp-atlas-search-index:
+
+====================
+Atlas Search Indexes
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: index, query, text search, efficiency
+
+Overview
+--------
+
+The MongoDB Atlas Search feature enables you to perform full-text
+searches on collections hosted on Atlas. Before you can perform Atlas
+Search queries, you must create indexes that specify which
+fields to index and how they are indexed.
+
+To learn more about Atlas Search, see the :atlas:`Atlas Search Overview
+</atlas-search/atlas-search-overview/>`.
+
+You can use the following methods on a ``mongocxx::search_index_view`` instance
+to manage your Atlas Search indexes:
+
+- ``create_one()``
+- ``create_many()``
+- ``list()``
+- ``update_one()``
+- ``drop_one()``
```

Comment:
S: A list on its own like this probably isn't very useful. I would suggest:

- Making each list entry a link to its subsection below;
- Adding a short description to each list entry; or
- Both!

Reply:
Did both, let me know if you think this is an improvement!